tests/unit_test/channels/mail_channel_test.py

```
@@ -0,0 +1,640 @@
+import asyncio
+import os
+from unittest.mock import patch, MagicMock
+
+import pytest
+from imap_tools import MailMessage
+
+from mongoengine import connect, disconnect
+
+from kairon import Utility
+os.environ["system_file"] = "./tests/testing_data/system.yaml"
```

Ruff (0.7.0)

Use capitalized environment variable `SYSTEM_FILE` instead of `system_file`

Replace `system_file` with `SYSTEM_FILE`

(SIM112)